src/arrays/arrays_with_methods/hard/q12/ArrayOfMultiples.java

```diff
@@ -0,0 +1,43 @@
+package arrays.arrays_with_methods.hard.q12;
+
+/*
+Write a Java program that asks the user to input an array of 10 integers then passes it to a method called
+multiplyElements. The method should return a new integer array containing the element stored in the original array
+multiplied by the preceding element, with the first element wrapping back around to be multiplied by the last. Ex.
+element 2 in the new array is element 2 * element 1 from the original array. The new array should be returned and
+printed in the main method. Note: your multiplyElements method should be general enough to work with any array size.
```

Comment:
For this, we should specify that first element should get multiplied by the last element if that is your intention